appservice/src/editScmType.ts

```diff
@@ -23,10 +23,18 @@ export async function editScmType(client: SiteClient, node: AzureTreeItem): Prom
         await connectToGitHub(node, client);
     } else {
         config.scmType = newScmType;
-        // to update one property, a complete config file must be sent
+        // to update one property, a complete config file must be sentnpm
         await client.updateConfiguration(config);
     }
     ext.outputChannel.appendLine(localize('deploymentSourceUpdated,', 'Deployment source for "{0}" has been updated to "{1}".', client.fullName, newScmType));
+    if (newScmType === ScmType.LocalGit) {
+        const user: WebSiteManagementModels.User = await client.getPublishingUser();
+        if (user.publishingUserName) {
+            // first time users must set up deployment credentials via the Portal or they will not have a UserName
+            const gitCloneUri: string = `https://${user.publishingUserName}@${client.gitUrl}`;
+            ext.outputChannel.appendLine(localize('gitCloneUri', 'Git Clone Uri for "{0}": "{1}"', client.fullName, gitCloneUri));
```

Comment:
Is https the only supported uri or could users use an ssh remote uri?

Reply:
As far as I've seen, every uri has been https for the git remote url, but I could be wrong about that.

Reply:
If it supports ssh I'd prefer if we printed that as well

Reply:
I only see SSH support if you deploy via DevOps: https://docs.microsoft.com/en-us/azure/devops/repos/git/use-ssh-keys-to-authenticate?view=vsts

But I don't think that'd be trivial to implement.